src/sympc/api.py

```diff
@@ -247,6 +247,10 @@
         "sympc.tensor.replicatedshare_tensor.ReplicatedSharedTensor.view",
         "sympc.tensor.replicatedshare_tensor.ReplicatedSharedTensor",
     ),
+    (
+        "sympc.tensor.replicatedshare_tensor.ReplicatedSharedTensor.reshape",
```

Comment:
Q: I don't think you need this, right?

Reply:
Not for this pr. I was just adding in general, because it will be used in future. I found the use when reshaping outputs for neural networks and thought I should add it.

Reply:
Could you remove it from here and we can add it when we need it?

Reply:
Alright.